src/backend/TrafficCourts/Workflow.Service/Mappings/Mapper.cs

```diff
@@ -14,8 +14,17 @@ public static EmailHistory ToEmailHistory(DisputantEmailSent src)
         target.ToEmailAddress = src.Message?.To is not null ? src.Message.To : "unknown";
         target.SuccessfullySent = EmailHistorySuccessfullySent.Y;
         target.Subject = src.Message?.Subject is not null ? src.Message.Subject : "unknown";
-        target.HtmlContent = src.Message?.HtmlContent is not null ? src.Message.HtmlContent : "";
```

Comment:
ords endpoint does some validation including that only one of htmlcontent or plaintextcontent can be non-null.  staff api was ensuring that both values were non-null thus the failure on every call